jina/orchestrate/deployments/config/helper.py

```diff
@@ -112,6 +114,8 @@ def construct_runtime_container_args(cargs, uses_metas, uses_with, pod_type):
         container_args.extend(['--uses-metas', json.dumps(uses_metas)])
     if uses_with is not None:
         container_args.extend(['--uses-with', json.dumps(uses_with)])
+    if env_from_secret is not None:
+        container_args.extend(['--env-from-secret', json.dumps(env_from_secret)])
```

Comment:
I do not think `env_from_secret` should be added to the `container`. This is an `orchestration` argument and should not be seen by the `runtime`